.docksal/commands/open

```diff
@@ -4,4 +4,4 @@
 ##
 ## Usage: fin open
 
-open https://${VIRTUAL_HOST}${NC}
+open //${VIRTUAL_HOST}${NC}
```

Comment:
Can we not keep it https always?   there are lots of external scripts and other drupal pieces that just assume we're on a secure site these days

Reply:
Updated.